src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -60,6 +60,13 @@
 
 
 @cloup.group(context_settings=CLICK_CTX_SETTINGS)
+@click.version_option(
+    None,  # version number not set, click will infer it using setuptools
+    "-v",
+    "--version",
+    prog_name="Orquestra SDK",  # displayed to the user (otherwise it's just `orq`)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I'm happy with `orq`, because `orq` is part of the package. But if we're going to use `Orquestra SDK`, please change it to `Orquestra Workflow SDK`
```suggestion
    prog_name="Orquestra Workflow SDK",  # displayed to the user (otherwise it's just `orq`)
```